src/server/src/main/java/io/cassandrareaper/storage/cassandra/Migration021.java

```diff
@@ -56,21 +56,23 @@ public static void migrate(Session session, String keyspace) {
                       + "'compaction_window_size': '2', "
                       + "'compaction_window_unit': 'MINUTES'}");
 
-          LOG.info("Altering {} to use TWCS...", METRICS_V2_TABLE);
-          session.execute(
-                  "ALTER TABLE " + METRICS_V2_TABLE + " WITH compaction = {'class': 'TimeWindowCompactionStrategy', "
-                      + "'unchecked_tombstone_compaction': 'true', "
-                      + "'compaction_window_size': '1', "
-                      + "'compaction_window_unit': 'HOURS'}");
+          if (session.getCluster().getMetadata().getKeyspace(keyspace).getTable(METRICS_V2_TABLE) != null) {
+            LOG.info("Altering {} to use TWCS...", METRICS_V2_TABLE);
+            session.execute(
+                    "ALTER TABLE " + METRICS_V2_TABLE + " WITH compaction = {'class': 'TimeWindowCompactionStrategy', "
+                        + "'unchecked_tombstone_compaction': 'true', "
+                        + "'compaction_window_size': '10', "
+                        + "'compaction_window_unit': 'MINUTES'}");
```

Comment:
why not just remove this?
ie, why do you need to alter METRICS_V2_TABLE to TWCS, in a codebase where it is now METRICS_V3_TABLE ?